release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -842,7 +842,7 @@ In the following tables, features are marked with the following statuses:
 
 |CSI Azure File Driver Operator
 |Technology Preview
-|General Availibility 
+|General Availibility
```

Comment:
```suggestion
|General Availability
```
Spelling

Reply:
This is actually something that I didn't change (this is only a whitespace removal) but I'm happy to fix while I'm in here!